docker_daemon/README.md

```diff
@@ -109,6 +112,28 @@ For more information about building custom Docker containers with the Datadog Ag
 
 The docker_daemon check is compatible with all major platforms
 
+## Agent v6 
+
+The new docker check is named `docker` and no longer `docker_daemon`. All features are ported, excepted the following deprecations:
```

Comment:
I'd rephrase to something like
> The new docker check is named `docker`. Starting from version 6.0, the Agent won't load the `docker_daemon` check anymore, even if it will be still available and maintained for Agent version 5.x.